### DISCUSSION TITLE
Is typeahead/autocomplete possible

### DISCUSSION THREAD

**Discussion by elie-h** [epicweb-dev/epic-stack#336] [Q&A]
Hi!

This might be specific to prisma/sqlite but wondering if building text search or autocomplete is possible? 
Has anyone built an example of this?

**Comment by kentcdodds** [comment]
Hi @elie-h,

We do have a search page: https://www.epicstack.dev/users

This does a case insensitive search on the username and name of users.

It's relatively naive. So for something more robust you may look into using https://sqlite.org/fts5.html

If that doesn't quite satisfy your use case then you probably want to bring in https://www.elastic.co/elasticsearch/

I hope that helps!
